lib/internal/event_target.js

```diff
@@ -463,6 +475,22 @@ class NodeEventTarget extends EventTarget {
     this.addEventListener(type, listener, { [kIsNodeStyleListener]: true });
     return this;
   }
+  emit(type, arg) {
+    if (arguments.length > 2) {
+      throw new ERR_INVALID_ARG_VALUE(
+        'arguments',
+        ArrayFrom(arguments).slice(2),
+        'Passing more than one argument to NodeEventTarget' +
+        ' dispatch is not supported'
```

Comment:
`NodeEventTarget` is purely internal, so I would not mention it here, tbh. Maybe just say `` `.emit()` on a Node.js `EventTarget` ``?